nsv13/code/modules/overmap/traders.dm

```diff
@@ -163,25 +183,23 @@
 /datum/trader/proc/give_mission(mob/living/user)
 	if(!isliving(user))
 		return
-	if(!mission_types.len || missions.len >= max_missions)
+		
+	var/list/valid_missions = list()
+	
+	for(var/m in missions) // Get all valid missions the crew qualifies for
+		var/datum/nsv_mission/mission = m
```

Comment:
```suggestion
	for(var/datum/nsv_mission/mission as() in missions) // Get all valid missions the crew qualifies for
```
Most efficient way to do a typeless for loop.

Reply:
travis didnt like it f

Reply:
Which is why I said you'd need to update SpacemanDMM.